product/views/Service.yaml

```diff
@@ -17,6 +17,9 @@ name: Service
 # Main DB table report is based on
 db: Service
 
+db_options:
+  :use_sql_view: false
```

Comment:
should this be true?  😕

Reply:
😮 I tweaked it to see if it did indeed speed things up... ugh